src/components/Home.tsx

```diff
@@ -17,11 +21,23 @@ const Title = styled.h1<HeadingProps>`
   ${typography}
 `;
 
+const LangButton = styled.button<HeadingProps>`
```

Comment:
we can actually change LangButton to:
```
const LangButton = styled.button``
```
since when we use it, we're not using any styling yet